docs/src/content/docs/book/cells.mdx

```diff
@@ -151,32 +151,48 @@ It affects the process of constructing and parsing cell values by causing them t
 
 In addition, the [TL-B][tlb] representation produced by Tact changes too:
 
-```tact {3-5, 8-10}
-contract SerializationExample {
-    // By default
-    cRef: Cell;    // ^cell in TL-B
-    bRef: Builder; // ^builder in TL-B
-    sRef: Slice;   // ^slice in TL-B
+```tact
+struct TwoCell {
+    cRef: Cell;              // ^cell in TL-B
+    cRem: Cell as remaining; // remainder<cell> in TL-B,
+                             // see explanation for those below
+}
 
-    // With `remaining`
-    cRem: Cell as remaining;    // remainder<cell> in TL-B
+struct TwoBuilder {
+    bRef: Builder;              // ^builder in TL-B
     bRem: Builder as remaining; // remainder<builder> in TL-B
-    sRem: Slice as remaining;   // remainder<slice> in TL-B
+}
+
+struct TwoSlice {
+    sRef: Slice;              // ^slice in TL-B
+    sRem: Slice as remaining; // remainder<slice> in TL-B
+}
+
+contract SerializationExample {
+    cell2: TwoCell;
+    builder2: TwoBuilder;
+    slice2: TwoSlice;
 
     // Constructor function,
     // necessary for this example contract to compile
     init() {
-        self.cRef = emptyCell();
-        self.bRef = beginCell();
-        self.sRef = emptySlice();
-        self.cRem = emptyCell();
-        self.bRem = beginCell();
-        self.sRem = emptySlice();
+        self.cell2 = TwoCell{
+            cRef: emptyCell(),
+            cRem: emptyCell(),
+        };
+        self.builder2 = TwoBuilder{
+            bRef: beginCell(),
+            bRem: beginCell(),
+        };
+        self.slice2 = TwoSlice{
+            sRef: emptySlice(),
+            sRem: emptySlice(),
+        };
     }
 }
 ```
 
-There, `^cell`, `^builder` and `^slice` in [TL-B][tlb] syntax mean the reference to [`Cell{:tact}`](#cells), [`Builder{:tact}`](#builders) and [`Slice{:tact}`](#slices) values respectively, while the `remainder<…>` of `cell`, `builder` or `slice` tells that the given value would be stored as a `Slice{:tact}` directly and not as a reference.
+There, `^cell`, `^builder` and `^slice` in [TL-B][tlb] syntax mean the reference to [`Cell{:tact}`](#cells), [`Builder{:tact}`](#builders) and [`Slice{:tact}`](#slices) values respectively, while the `remainder<…>` of `cell`, `builder` or `slice` tells that the given value would be stored as a `Cell{:tact}`, `Builder{:tact}` or `Slice{:tact}` directly and not as a reference. Note that the `remainder<X>` is not part of the regular [TL-B][tlb] syntax and is only present in Tact-generated comments in various [build artifacts](/book/compile#artifacts) as a visual clue.
```

Comment:
And I still don't get what it means: this change does not seem to be an explanation.